code/src/components/TodoSummary.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import styled from 'styled-components'
+import { Title, Subtitle } from 'library/Text'
+
+export const TodoSummary = () => {
+  const items = useSelector((store) => store.todos.items)
+  const completedItems = items.filter((item) => item.complete)
+
+  if (items.length === 0) {
+    return ( 
+    <HeaderContainer> 
+      <Title>No todos added yet! Add todos below:</Title>
+    </HeaderContainer>
+    )
+  }
+
+  return (
+    <HeaderContainer>
+      <Title>On your todo-list today, you have {items.length} todo{items.length > 1 ? 's.' : '.'}</Title>
+      {completedItems.length === items.length 
+      ? <Subtitle>Well done, you have completed all your tasks!</Subtitle>
+      : <Subtitle>You have completed {completedItems.length} todo{completedItems.length === 1 ? '.' : 's.'}
```

Comment:
I would split this up on several lines: 
<Subtitle>
          You have completed {completedItems.length} todo
          {completedItems.length === 1 ? "." : "s."}{" "}
          {items.length - completedItems.length} more to go!
        </Subtitle>